Gemfile

```diff
@@ -5,12 +5,9 @@ gemspec
 gem 'danger', '~> 8.0'
 gem 'danger-rubocop', '~> 0.6'
 
-group :test do
-  gem 'codecov', require: false
-  gem 'rspec'
-  gem 'webmock', require: false, group: :test
-  gem 'yard'
-end
+gem 'codecov', require: false
+gem 'webmock', require: false
+gem 'yard'
```

Comment:
This LGTM. Just curious, though, did you decide to remove the `test` group because:

- Everyone running `bundle install` on this project is expected to run the tests; and
- Consumer of the gem get the dependencies via what's specified in the `.gemspec`, so there's no need to add groups here because, if we don't add these to the `.gemspec` they won't be part of the gem dependency resolution chain

?